package.json

```diff
@@ -36,7 +36,7 @@
     "release:stable:npm-failed": "yarn lerna publish from-package -y --loglevel verbose -m 'chore(release): publish' --conventional-commits",
     "release:storybook": "yarn build:storybook && gh-pages -d docs",
     "version": "node ./tools/build/version.js",
-    "clean": "node ./tools/build/clean-repo.js && lerna clean --yes && lerna run clean && yarn && yarn bootstrap",
+    "clean": "node ./tools/build/clean-repo.js && lerna clean --yes && lerna run clean && rm -rf node_modules/ && yarn && yarn bootstrap",
```

Comment:
Had all kinds of issues with babel and its plugins, based on the order in which they are installed and giving variable build behaviors locally and in CI. Being able to clean _everything_ was very useful

Reply:
How do you feel about moving this to a `clean:full` command instead?

Reply:
I'm in